app/views/pages/home.html.erb

```diff
@@ -1,2 +1,32 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+ <a name="about"></a>
+ <div class="intro-header">
+     <div class="container">
+
+         <div class="row">
+             <div class="col-lg-12">
+                 <div class="intro-message">
+                     <h1>Taste of code</h1>
+                     <h3>Beginner coding workshop</h3>
+                     <hr class="intro-divider">
+                     <ul class="list-inline intro-social-buttons">
+                         <li>
+                             <a <%= link_to "JOIN TOC", attend_path, class: "btn btn-default btn-lg" %></i> <span class="network-name"></span></a>
+                         </li>
+                         <li>
+                             <a <%= link_to "INCOMPANY", in_company_path, class: "btn btn-default btn-lg" %></i> <span class="network-name"></span></a>
+                         </li>
+                         <li>
+                             <a <%= link_to "HOST TOC", sponsor_path, class: "btn btn-default btn-lg" %></i> <span class="network-name"></span></a>
+
+                         </li>
+                     </ul>
+                 </div>
+             </div>
+         </div>
+
+     </div>
+ </div>
+
+ <script src="js/jquery.js"></script>
+
+ <script src="js/bootstrap.min.js"></script>
```

Comment:
These two scripts don't belong in there.

I have the feeling that you are not following the instructions to setup bootstrap in your project as in the Reader.
Please review [this article](https://reader.codaisseur.com/courses/intermediate-bootcamp-1d45fcf7-b918-482f-a526-e126d3c063d9/57/codaisseurbnb-project-setup/installing-bootstrap).

If you do all the steps in there, you will have a proper application working with bootstrap 😄